### ISSUE TITLE
Output name conflict on multiple async call

### ISSUE THREAD

**Issue by leonamjlpaula** [bamlab/react-native-image-resizer#231]
When creating multiple resized images with Promise.all() I got an "File already exists" exception for some Android devices in my beta testers. Would be great to have an output name param.

**Comment by johanschram** [comment]
I do have the same problem when using Promise.all()

**Comment by namthan** [comment]
I'm facing the same issue.

**Comment by sitorhy** [comment]
`ImageResizerModule.java`
"Long.toString(new Date().getTime())" will get duplicate values when concurrent tasks perform

e.g
```
for(int i=0;i<10;i++){
     System.out.println(Long.toString(new Date().getTime()));
}
```

**Comment by linoleum00** [comment]
Hi, I was wondering, why is it that they didn't use the same approach in Java, as they did on ios, where they use a UUID for the file name instead of the date in milliseconds? Just saying...

**Comment by PierreCapo** [comment]
Fixed in v1.4.2. See https://github.com/bamlab/react-native-image-resizer/pull/269 for more information
